tests/test_v2_event.py

```diff
@@ -18,6 +18,7 @@ def v2_payload_no_data(self):
                 "id": "evt_234",
                 "object": "v2.core.event",
                 "type": "financial_account.balance.opened",
+                "livemode": "false",
```

Comment:
```suggestion
                "livemode": False,
```

Same in the other method. Maybe change that one to True